### PR TITLE
Update _index.en.md

### PR DESCRIPTION
**Pull request by dinger1986**
changed install scripts slightly

### CODE REVIEW

File: content/self-host/client-deployment/_index.en.md [hunk 1]
```diff
@@ -114,6 +114,9 @@ rustdesk.exe --silent-install
 timeout /t 20
 
 cd "C:\Program Files\RustDesk\"
+rustdesk.exe --install-service -wait -Verbose
```
Comment (rustdesk) [inline content/self-host/client-deployment/_index.en.md:117]: Are you sure this works?

Comment (rustdesk) [reply content/self-host/client-deployment/_index.en.md:117]: I believe `-wait -Verbose` does nothing here.

Comment (dinger1986) [reply content/self-host/client-deployment/_index.en.md:117]: it does install the service yes. you are right sorry just copied it from notes from helping someone earlier and didnt check

Comment (dinger1986) [reply content/self-host/client-deployment/_index.en.md]: changed now

**Pull request merged**